src/api_client.rs

```diff
@@ -494,10 +494,18 @@ impl Client {
     }
 
     pub async fn get_sync_committee_duties(
+        &self,
         epoch: Epoch,
         indices: &[ValidatorIndex],
-    ) -> Result<(Root, Vec<SyncCommitteeDuty>), Error> {
-        unimplemented!("")
+    ) -> Result<Vec<SyncCommitteeDuty>, Error> {
+        let endpoint = format!("eth/v1/validator/duties/sync/{epoch}");
+        let result: Value<Vec<SyncCommitteeDuty>> = self
+            .http_post(&endpoint, indices)
+            .await?
+            .json()
```

Comment:
this by itself is good but I would like to address this: https://github.com/ralexstokes/beacon-api-client/pull/50#discussion_r1003764461

and it will affect this implementation